source/channels/make-calls.rst

```diff
@@ -69,29 +76,34 @@ Media (audio/video) is encrypted using security standards as part of WebRTC. It'
+The only external service used is Mattermost official STUN server (``stun.global.calls.mattermost.com``) which is configured as default. This is primarily used to find the public address of the Mattermost server. The only information sent to this service is the IP addresses of clients connecting as no other traffic goes through it. It can be removed in the System Console if you want to provide an ``ICE Host Override`` setting instead.
 
 Troubleshooting
 ---------------
 
+My audio doesn't work when I join a call
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you can hear the other participants in the call but they can't hear you, select the horizontal dots next to the call end button in the widget. From there, you can check and change your audio output and microphone settings. Select the horizontal dots again to close the menu.
+
 My call is disconnected after a few seconds
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-My call is disconnected after a few seconds and I can't transmit voice nor hear anything.
-
-This is usually a sign that the underlying UDP channel has not been established and the connection times out after ~10 seconds. When the connection has been established correctly an ``rtc connected`` line should appear in the client-side logs (JS console). There isn't a single solution as it depends on the infrastructure/deployment specifics. However, if you're a system or network admin, you may need to open up the UDP port or configure the network accordingly.
+This is usually a sign that the underlying UDP channel has not been established and the connection times out after ~10 seconds. When the connection has been established correctly an ``rtc connected`` line should appear in the client-side logs (JS console). There isn't a single solution as it depends on your infrastructure/deployment specifics. However, if you're a system or network admin, you may need to open up the UDP port or configure the network accordingly.
 
 I can't screen share using Mattermost desktop on macOS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There is a known bug on macOS with some versions of Chrome (which is used by Mattermost desktop). If you've given screen sharing permissions to Mattermost desktop, and are still unable to screen share, do the following:
+There's a known bug on macOS with some versions of Chrome (which is used by Mattermost desktop). If you've given screen sharing permissions to Mattermost desktop, and are still unable to screen share, do the following:
 
 1. Quit Mattermost.
 2. Open Terminal.
 3. In terminal, run: ``tccutil reset ScreenCapture Mattermost.Desktop``
 4. Restart Mattermost and start a call.
 5. Select **Screen share** and give it permissions again.
-6. Restart Mattermost.
+6. Restart Mattermost again.
+
+If the issue persists please post in the public calls channel.
 
 Debugging
```

Comment:
@streamer45, I believe this should also be moved to the deployment section - thoughts?